examples/creating_printing_table/creating_printing_table.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015 Steinwurf ApS
+// All Rights Reserved
+//
+// Signed-off-to-Steinwurf-by: Juan Cabrera <jcabre14@student.aau.dk>
+// Distributed under the "BSD License". See the accompanying LICENSE.rst file.
+
+
+#include <fstream>
+#include <cstdint>
+
+#include <tables/table.hpp>
+#include <tables/csv_format.hpp>
+#include <tables/json_format.hpp>
+#include <tables/python_format.hpp>
+
+/// @example creating_printing_table.cpp
+///
+/// Simple example showing how to create a table
+/// and print it in any desired format.
+
+int main()
+{
+
+    // In this example we will generate 100 random numbers. We then
+    // will store those numbers in a table with two columns:
+    // The random number and the iteration in wich it was generated.
+
+    // We set the names for the ouput files.
+    std::string out_csv = "out.csv";
+    std::string out_json = "out.json";
+    std::string out_python = "out_python.json";
```

Comment:
It's also kinda weird i must admit, also I don't think it's used anymore. But it's a representation of the data as a python dictionary. It looks very similar to the json format but there's a few differences. For instance in json a true value is ``true``, but in python it's ``True``.